workers/task-driver/src/driver.rs

```diff
@@ -158,7 +160,7 @@ impl TaskExecutor {
         // Build a runtime
         let runtime = TokioRuntimeBuilder::new_multi_thread()
             .enable_all()
-            .worker_threads(TASK_DRIVER_N_THREADS)
+            // .worker_threads(TASK_DRIVER_N_THREADS)
```

Comment:
dead code

Reply:
ah yeah, that's from my testing - but fwiw i think we should remove this and let tokio select the number of threads based on underlying available parallelism. thoughts?